frameworks/cassandra/tests/test_sanity.py

```diff
@@ -125,3 +125,33 @@ def test_custom_jmx_port() -> None:
     for task in tasks:
         _, stdout, _ = sdk_cmd.run_cli("task exec {} lsof -i :7200".format(task.id))
         assert expected_open_port in stdout
+
+
+@pytest.mark.sanity
+def test_udf() -> None:
+    test_jobs: List[Dict[str, Any]] = []
+    try:
+        test_jobs = config.get_udf_jobs(node_address=config.get_foldered_node_address())
+        # destroy/reinstall any prior leftover jobs, so that they don't touch the newly installed service:
+        for job in test_jobs:
+            sdk_jobs.install_job(job)
+
+        new_config = {
+            "cassandra": {
+                "enable_user_defined_functions": True,
+                "enable_scripted_user_defined_functions": True,
+            }
+        }
+        sdk_service.update_configuration(
```

Comment:
Sorry I wasn't clear but by changing the test to update configs you're still effectively relaunching all the tasks in the service but I reconsidered my request. Because we don't install UDFs by default we _should_ follow this pattern and update configs and test the feature like you are doing.